Frontend/ui-driver/src/MerchantConfigs/CommonConfig.purs

```diff
@@ -34,4 +34,7 @@ commonConfig =
         , showCorporateAddress: false
         , imageUploadOptional: true
         , autoPayBanner: true
+        , black800 : "#454545"
```

Comment:
Why colors are added to configs?

Reply:
these 3 colors are used for different merchant in different color in customer side. because of that in driver side also moved to configs.